src/omdata.h

```diff
@@ -22,6 +22,27 @@ struct overmap_spawns {
     int max_population;
     int chance;
 };
+ //terrain flags enum! this is for tracking the indices of each flag.
+    //is_asphalt, is_building, is_subway, is_sewer, is_ants,
```

Comment:
They are listed *again* right below it as part of the enum definition, why *repeat* them?

If one can read them in the comment, one can also read them in the actual code. Do you really think someone sees them in the comment but is unable to find them in the actual definition? And I usually use the search-in-document function (or similar) of the editor when I need to find some definition.